addons/kotsadm/alpha/install.sh

```diff
@@ -186,6 +182,25 @@ function kotsadm_api_patch_prometheus() {
     cp "$DIR/addons/kotsadm/alpha/patches/api-prometheus.yaml" "$DIR/kustomize/kotsadm/api-prometheus.yaml"
 }
 
+function kotsadm_metadata_configmap() {
+    local src="$1"
+    local dst="$2"
+
+    # Create an empty application-metadata.yaml to keep kustomization happy
```

Comment:
Is there a reason we should include empty files instead of not including the file in the kustomization.yaml?

Reply:
I found kustomization will fail because its expecting kotsadm-application-metadata.yaml (see kustomization.yaml file).  So in the cases where there's no branding, I needed to keep kustomization happy with this empty file (as the comment implies).

Reply:
I decided to no longer use this empty file, instead i'm using insert_resources function to insert a kustomization resource for the application-metadata (if it exists).  See my latest commit.